Backend/controllers/StatisticsResourceController.js

```diff
@@ -32,46 +31,41 @@ exports.getStatisticsResourceById = async(req, res) => {
 };
 
 exports.getStatisticsResourceByFields = async(req, res) => {
-    try {
-        if (req.query.language != null && req.query.title != null) {
-            let statisticsResource = await StatisticsResource.find({ language: req.query.language, title: req.query.title });
-            res.status(200).send(statisticsResource);
-        } else if (req.query.language != null) {
-            let statisticsResource = await StatisticsResource.find({ language: req.query.language });
-            res.status(200).send(statisticsResource);
-        } else if (req.query.title != null) {
-            let statisticsResource = await StatisticsResource.find({ title: req.query.title,language:"English" });
-            res.status(200).send(statisticsResource);
-        } else {
-            let statisticsResource = await StatisticsResource.find({language:"English"});
-            res.status(200).send(statisticsResource);
-        }
 
-    } catch (error) {
-        res.status(500).send(error);
+    if (req.query.language != null && req.query.title != null) {
+        let statisticsResource = await StatisticsResource.find({ language: req.query.language, title: req.query.title });
+        res.status(200).send(statisticsResource);
+    } else if (req.query.language != null) {
+        let statisticsResource = await StatisticsResource.find({ language: req.query.language });
+        res.status(200).send(statisticsResource);
+    } else if (req.query.title != null) {
+        let statisticsResource = await StatisticsResource.find({ title: req.query.title,language:"English" });
+        res.status(200).send(statisticsResource);
+    } else {
+        let statisticsResource = await StatisticsResource.find({language:"English"});
+        res.status(200).send(statisticsResource);
```

Comment:
If we're editing this, would be nice to have the if statements create a filter then call find at one place instead of all these calls. What do you think?